yt/frontends/cf_radial/data_structures.py

```diff
@@ -0,0 +1,239 @@
+"""
+CF Radial data structures
+
+
+
+"""
+
+import os
+import stat
+import weakref
+
+import numpy as np
+
+from yt.data_objects.grid_patch import AMRGridPatch
```

Comment:
Looks like this module changed locations since the initial PR: 
```suggestion
from yt.data_objects.index_subobjects.grid_patch import AMRGridPatch
```